stl/inc/array

```diff
@@ -834,7 +834,7 @@ _NODISCARD constexpr array<remove_cv_t<_Ty>, _Size> to_array(_Ty (&_Array)[_Size
                                     "to_array does not accept multidimensional arrays.");

```

Comment:
Shouldn't we qualify the call to `_STD _Swap_ranges_unchecked` on line 436?

Reply:
`_Swap_ranges_unchecked` itself needs ADL for the element type (to determine which `swap` is to be called)...

I'm adding `_STD` now. But I haven't found any case that can be made to work by qualifying `_Swap_ranges_unchecked`.

Reply:
1. IMO, it's better for the ADL to occur in the unqualified `swap` call than a call to `_Some_ugly_internal_name` so potential error messages point directly to `swap`. It may help users understand the problem a bit quicker.
2. For readers, it's better to consistently qualify all calls to namespace-scope function( template)s except for a very small number of well-known exceptions (which we nevertheless comment with `// intentional ADL`). I can immediately see that `_STD _Some_ugly_internal_name` is fine without knowing the internals of `_Some_ugly_internal_name` .